packages/dev/docs/pages/releases/2022-05-27.mdx

```diff
@@ -0,0 +1,49 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {BlogPostLayout, Hero} from '@react-spectrum/docs';
+export default BlogPostLayout;
+
+---
+description: React 18 is now officially supported! Other updates include a major version bump to our workflow icons, support for emphasized Tabs, and various bug fixes to our existing components and hooks. Thank you to all who contributed to this release!
+date: 2022-05-27
+---
+
+# May 27, 2022 Release
+
+Ever since React 18 was announced last July, compatibility has been front of mind for the team. We are extremely pleased to announce that React 18 is now officially supported with our library! Special thanks to [@eps1lon](https://github.com/eps1lon) for his
+help with getting our tests working with React 18.
```

Comment:
Open to feedback on wording here

Reply:
I think it's good.

One thought, should the portion on Spectrum 18 and RSP library compatibility be flipped so it's more like “our library officially supports Spectrum 18.”

Reply:
```suggestion
Ever since React 18 was announced last July, compatibility has been front of mind for the team. We are extremely pleased to announce that React 18 is now officially supported by our library! Special thanks to [@eps1lon](https://github.com/eps1lon) for his
help with getting our tests working with React 18.
```